qiskit/quantum_info/states/statevector.py

```diff
@@ -492,7 +492,7 @@ def _expectation_value_pauli(self, pauli, qargs=None):
 
     def expectation_value(
         self, oper: BaseOperator | QuantumCircuit | Instruction, qargs: None | list[int] = None
-    ) -> complex:
+    ) -> numpy.complex128:
```

Comment:
You may need to update "Returns" part of docstring as well.
You have an option to return `np.dot(conj.data, val.data).item()`. In this case, you don't have to update type hint and docstring.

Reply:
Thanks. This comes from my local `mypy` error, but should be fixed in another PR. I'll revert this line.